Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -105,27 +125,26 @@ def __init__(self):
         )
         self.inputsBox.setEnabled(bool(self.useidentifiers))
 
-        for i in range(5):
-            box = gui.vBox(self.inputsBox, "Dataset #%i" % (i + 1),
-                           addSpace=False)
-            box.setFlat(True)
-            model = itemmodels.VariableListModel(parent=self)
-            cb = gui.OrangeComboBox(
-                minimumContentsLength=12,
-                sizeAdjustPolicy=QComboBox.AdjustToMinimumContentsLengthWithIcon
+        box = gui.vBox(self.inputsBox, 'feature', addSpace=False)
+        box.setFlat(True)
+        model = itemmodels.VariableListModel(parent=self)
+        self.cb = cb = gui.OrangeComboBox(
+            minimumContentsLength=12,
+            sizeAdjustPolicy=QComboBox.AdjustToMinimumContentsLengthWithIcon
```

Comment:
I agree, this is clearer. The referenced issue in this pull request has a sketch by @BlazZupan with "Use identifiers", but i think that's because of historical reasons.